ci-operator/step-registry/baremetalds/assisted/operator/ztp/baremetalds-assisted-operator-ztp-ref.yaml

```diff
@@ -12,28 +12,5 @@ ref:
   - name: DISCONNECTED
     default: "false"
     documentation: Should we work on top of a disconnected environment
-  - name: ASSISTED_NAMESPACE
-    default: "assisted-installer"
-  - name: ASSISTED_CLUSTER_NAME
-    default: "assisted-test-cluster"
-  - name: ASSISTED_CLUSTER_DEPLOYMENT_NAME
-    default: "assisted-cluster-deployment"
-  - name: ASSISTED_INFRAENV_NAME
-    default: "assisted-infraenv"
-  - name: ASSISTED_PRIVATEKEY_NAME
-    default: "assisted-ssh-private-key"
-  - name: ASSISTED_PULLSECRET_JSON
-    default: "/root/dev-scripts/pull_secret.json"
-  - name: ASSISTED_PULLSECRET_NAME
-    default: "assisted-pull-secret"
-  - name: ASSISTED_OPENSHIFT_VERSION
-    default: "openshift-v4.8.0"
-  dependencies:
-  - name: "release:latest"
-    env: ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE
-  - name: "release:latest"
-    env: OPENSHIFT_INSTALL_RELEASE_IMAGE
```

Comment:
These two variables are not redundant. It was split because we wanted to use different versions of OCP for these two things.

OPENSHIFT_INSTALL_RELEASE_IMAGE = devscripts-setup installed image (hub cluster)
ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE = image installed by assisted installer (spoke cluster)

Reply:
The rest of the variables I suppose they have some default values in the scripts now. But we will want a way to override these.

Reply:
`ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE` was moved to `operator-setup` step
`OPENSHIFT_INSTALL_RELEASE_IMAGE` is used as part of `devscripts-setup` step

Regarding the rest of the variables - they are corrently not required and just causing confusion (since they had wrong values). In case it would be needed - they'll be added once again.

Reply:
I'm not sharing the need to remove all those variables
Just make sure we can still override ``ASSISTED_OPENSHIFT_VERSION`` and ``ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE`` (sure, it's on another step), the other ones are mainly names and constants
And keep in mind it means we now limit ourselves to one spoke installation per job, which is fine by me but still a limitation

Reply:
That's a good point. But as it is not supported I don't see any reason to keep it for now.